modules/ossm-deploying-automatic-gateway-injection.adoc

```diff
@@ -0,0 +1,81 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/v2x/ossm-traffic-manage.adoc
+
+:_content-type: PROCEDURE
+[id="ossm-deploying-automatic-gateway-injection_{context}"]
+= Deploying automatic gateway injection
+
+When deploying a gateway, you must opt-in to injection by adding an injection label or annotation to the gateway `deployment` object. The following example  `ConfigMap` object deploys a gateway with automatic injection.
+
+.Prerequisites
+
+* The namespace must be a member of the mesh by defining it in the `ServiceMeshMemberRoll` or by creating a `ServiceMeshMember` resource.
+
+.Example deployment with annotations
+[source,yaml]
+----
+  apiVersion: v1
+  kind: Service
+  metadata:
+    name: istio-ingressgateway
+    namespace: istio-ingress
+  spec:
+    type: LoadBalancer
+    selector:
+      istio: ingressgateway
+    ports:
+    - port: 80
+      name: http
+    - port: 443
+      name: https
+  ---
+  apiVersion: apps/v1
+  kind: Deployment
+  metadata:
+    name: istio-ingressgateway
+    namespace: istio-ingress
+  spec:
+    selector:
+      matchLabels:
+        istio: ingressgateway
+    template:
+      metadata:
+        annotations:
+          inject.istio.io/templates: gateway <1>
+        labels:
+          istio: ingressgateway <2>
+          sidecar.istio.io/inject: "true" <3>
+      spec:
+        containers:
+        - name: istio-proxy
+          image: auto <4>
+  ---
+  apiVersion: rbac.authorization.k8s.io/v1 <5>
+  kind: Role
+  metadata:
+    name: istio-ingressgateway-sds
+    namespace: istio-ingress
+  rules:
+  - apiGroups: [""]
+    resources: ["secrets"]
+    verbs: ["get", "watch", "list"]
+  ---
+  apiVersion: rbac.authorization.k8s.io/v1
+  kind: RoleBinding
+  metadata:
+    name: istio-ingressgateway-sds
+    namespace: istio-ingress
+  roleRef:
+    apiGroup: rbac.authorization.k8s.io
+    kind: Role
+    name: istio-ingressgateway-sds
+  subjects:
+  - kind: ServiceAccount
+    name: default
+----
+<1> Select the gateway injection template rather than the default sidecar template.
```

Comment:
Move the annotations to the correct code block and restart the numbering from 1 for each code block.

Reply:
Created OSSMDOC-728 to evaluate breaking the code block apart versus creating numbered steps. If we break, then steps might make more sense. Will follow-up after the release deadline.